mod-orders/mod-orders.postman_collection.json

```diff
@@ -4311,6 +4312,187 @@
 							},
 							"response": []
 						},
+						{
+							"name": "Edit piece by id",
+							"event": [
+								{
+									"listen": "prerequest",
+									"script": {
+										"id": "eb2052a4-e388-4860-80be-ea96976bcf21",
+										"exec": [
+											"let utils = eval(globals.loadUtils);",
+											"",
+											"// After changing order from pending to open in previous request, piece(s) is created",
```

Comment:
Taking into account that now we have `POST /orders/pieces`, PUT and DELETE `/orders/pieces/{id}` endpoints, separate folder with such requests can be created.